src/components/LeftSidebar/ConversationsList/ConversationSearchResult.vue

```diff
@@ -66,14 +62,17 @@ export default {
 	emits: ['click'],
 
 	setup(props) {
-		const { item, exposeMessages } = toRefs(props)
+		const { item } = toRefs(props)
 		const selectedRoom = inject('selectedRoom', null)
-		const { counterType, conversationInformation } = useConversationInfo({ item, exposeMessages })
+		const exposeDescription = inject('exposeDescription', false)
+		const exposeMessages = inject('exposeMessages', false)
```

Comment:
Did you provide it anywhere? _(RoomSelector or OpenConversationsList)_

Reply:
no, the prop wasn't used too. I will remove it as we don't intend to expose messages in `ConversationSearchResult` .